pages/linux/debsecan.md

```diff
@@ -0,0 +1,24 @@
+# debsecan
+
+> A tool to list vulnerabilities on a particular Debian installation.
+> More information: <https://gitlab.com/fweimer/debsecan>.
+
+- List vulnerable installed packages on the current host:
+
+`debsecan`
+
+- List vulnerable installed packages of a specific suite:
+
+`debsecan --suite {{release_code_name}}`
+
+- List only fixed vulnerabilities of testing ("sid"):
```

Comment:
Isn't testing `bookworm` and `sid` unstable right now? Also does `--suite` not support passing `testing` or `stable`? Because this way it breaks every 3 years.

Reply:
I thought Debian testing was always called sid.

Reply:
I find it confusing (`sid` seems to be [the code name of "unstable Debian"](https://wiki.debian.org/DebianUnstable))  
```suggestion
- List only fixed vulnerabilities:
```

Reply:
@pixelcmtd is right, `unstable` always has the *codename* `sid` from [Debian Wiki](https://wiki.debian.org/DebianUnstable). And [`testing`](https://wiki.debian.org/DebianTesting) always uses the *codename* of next stable release (that's `bookworm` as of now).

I mixed it up with `testing` because testing has no security updates and I'm accumstomed to use `sudo apt -t sid install {{package_name}}` to get packages from `unstable`.

> Also does --suite not support passing testing or stable? Because this way it breaks every 3 years.

No, it supports only *codename* like `bullseye`, `bookworm`, `sid` etc. The [man page](https://manpages.debian.org/testing/debsecan/debsecan.1.en.html) of `debsecan` also reads "The release code name has to be  used  ("sid"),  not the temporal name ("unstable")."